prometheus/getmessages_prometheus.py

```diff
@@ -846,6 +861,99 @@ def initialize_api_post_data(logger, if_config_vars):
     return to_send_data_dict
 
 
+def check_project_exist(logger, if_config_vars):
+    is_project_exist = False
+    try:
+        logger.info('Starting check project: ' + if_config_vars['project_name'])
+        # execute sql string
+        params = {
+            'operation': 'check',
+            'userName': if_config_vars['user_name'],
+            'licenseKey': if_config_vars['license_key'],
+            'projectName': if_config_vars['project_name'],
+        }
+        url = urllib.parse.urljoin(if_config_vars['if_url'], 'api/v1/check-and-add-custom-project')
+        response = send_request(logger, url, 'POST', data=params, verify=False, proxies=if_config_vars['if_proxies'])
+        if response == -1:
+            logger.error('Check project error: ' + if_config_vars['project_name'])
+        else:
+            result = response.json()
+            if result['success'] is False or result['isProjectExist'] is False:
+                logger.error('Check project error: ' + if_config_vars['project_name'])
+            else:
+                is_project_exist = True
+                logger.info('Check project success: ' + if_config_vars['project_name'])
+
+    except Exception as e:
+        logger.error(e)
+        logger.error('Check project error: ' + if_config_vars['project_name'])
+
+    create_project_sucess = False
+    if not is_project_exist:
+        try:
+            logger.info('Starting add project: ' + if_config_vars['project_name'])
+            # execute sql string
+            params = {
+                'operation': 'create',
+                'userName': if_config_vars['user_name'],
+                'licenseKey': if_config_vars['license_key'],
+                'projectName': if_config_vars['project_name'],
```

Comment:
Does the API support passing in system name? If so, we should add it to the config.ini file and send it too

Reply:
Agent add param: systemName, wait backend.